tests/phpunit/CRM/Utils/Mail/EmailProcessorTest.php

```diff
@@ -242,6 +248,7 @@ public function setUpDoNotCreateContact() {
         'domain' => 'example.com',
         'is_default' => '0',
         'is_contact_creation_disabled_if_no_match' => TRUE,
+        'is_non_case_email_skipped' => FALSE,
```

Comment:
This test wasn't actually testing anything before as it was skipping the non-case email.